tools/update-deps.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -xe
+
+source $(dirname $0)/common.sh
+
+update_dep_from_pypi alembic 1.0.11
```

Comment:
Interested to know why we can't use pip here, e.g clone the project repo, then do `pip install .`

I know it's not in RHEL by default, but for upstream use that would be OK and better aligned with typical OpenStack install-from-source usage e.g devstack?

Reply:
The idea is to treat both images the same, and only vary them based on the inputs to the source code where we actually need to. That's what we do for go.